src/main/java/minicraft/screen/OptionsMainMenuDisplay.java

```diff
@@ -20,6 +24,14 @@ public OptionsMainMenuDisplay() {
         );
     }
 
+    @Override
+	public void render(Screen screen) {
+		super.render(screen);
+        
+		// Warning users about screen resoluton requiring a restart
+		Font.drawCentered("Some settings may require a restart", screen, Screen.h-20, Color.RED);
+    }

```

Comment:
This isn't good enough. Ideally we would do the same as with the dialog box that appears when resetting keybinds.